modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/edit_form_view.jsp

```diff
@@ -36,10 +36,10 @@ String componentId = renderResponse.getNamespace() + "dataLayoutBuilder";
 	/>
 </aui:form>
 
-<aui:script require='<%= npmResolvedPackageName + "/js/pages/form-view/EditFormViewApp.es as App" %>'>
+<aui:script require='<%= npmResolvedPackageName + "/js/pages/form-view/EditFormViewApp.es as EditFormViewApp" %>'>
 	Liferay.componentReady('<%= componentId %>').then(
 		function(dataLayoutBuilder) {
-			App.default(
+			EditFormViewApp.default(
```

Comment:
Changed this for consistency (so that the name of the module export matches the name of the file).

One thing I noticed about this file, @brunobasto, is that the object with the `dataLayoutBuilder` is not being used at all (in fact `EditFormViewApp.default()` doesn't even take a second parmeter). I left that untouched on the assumption that work is ongoing here and you're probably going to use it in the future, right?

Reply:
@wincent yes, we are going to use it, thanks!

Reply:
Hey @wincent, thx for catching that. Yeah, this is an ongoing story and I'm sending to BChan in small chunks.